### ISSUE TITLE
USER ISSUE: cannot login to server after managing deed

### ISSUE THREAD

**Issue by mbrenn07** [StrangeLoopGames/EcoIssues#25039]
\
**User Id: (69988065-53e1-45fa-c864-08dcbd8319be)**
**Version:** 0.11.0.0 beta release-693

**Steps to Reproduce:**
1. Log into server

**Expected behavior:**
It should have logged in successfully

**Actual behavior:**
The following error message popped up:
```
Server encountered an exception:

<size=60.00%>Exception: NullReferenceException
Message:Object reference not set to an instance of an object.
Source:Anonymously Hosted DynamicMethods Assembly

System.NullReferenceException: Object reference not set to an instance of an object.
   at lambda_method64057(Closure, IMvcNetClient, Object)
   at Eco.Core.Controller.ControllerPacker.PackageMember(BsonSerializationContext& context, IController controller, ISyncMemberInfo memberInfo)
Outer Exceptions:
Exception has been thrown by the target of an invocation.
Cannot Convert Argument Type: Eco.Gameplay.Property.Deed to BSON.

Error during packaging property 'Accessors' of 'Eco.Gameplay.Property.Deed'
Error during packaging property 'HomesteadDeed' of 'Eco.Gameplay.Players.User'
Error during packaging property 'Ownables' of 'Eco.Gamep</size>...

Check log file to see full message.
```

**Do you have mods installed? Does the issue happen when no mods are installed?:**
No

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by SlayksWood** [comment]
@mbrenn07 Hi, thank you for the report, can you provide save file and log files?

**Comment by SlayksWood** [comment]
Internal number: ECO-36084

**Comment by mbrenn07** [comment]
Here are my log and .eco files: 
[Player.log](https://github.com/user-attachments/files/16638865/Player.log)
[player.txt](https://github.com/user-attachments/files/16638873/player.txt)

Right before this bug occurred I was attempting to unclaim some plots within my deed, but I was getting the error "This plot is already part of [my homestead]" when **unclaiming**, and wasn't able to unclaim any. This bug was completely preventing me from logging into the server when I submitted it yesterday, but I've just retried and it seems to be working again.

**Comment by mbrenn07** [comment]
Some additional info which may be helpful, I have one other friend added as a citizen to the deed and as a deed member.

**Comment by D3nnis3n** [comment]
This issue was fixed in Hotfix 11.0.1.
